Source/GSAutoLayoutVFLParser.m

```diff
@@ -0,0 +1,526 @@
+#import "GSAutoLayoutVFLParser.h"
```

Comment:
Same thing here.  Mainly the formatting is wrong.  Normally we go for the left brace on the next line and indented by two spaces instead of on the same line.   The algorithms here look correct, but I have not had a chance to test them in a running app.  I will do that tonight.   Also, we need to have the LGPL text on the top of both the source and the header files.

Reply:
BTW, I got your paperwork, thank you.

Reply:
Added the copyright notices